lib/systems/src/TorqueControllers.cpp

```diff
@@ -153,5 +153,103 @@ void TorqueControllerLoadCellVectoring::tick(
         {
             writeout_.command = TC_COMMAND_NO_TORQUE;
         }
+    }
+}
+
+void TorqueControllerSimpleLaunch::tick(
+    const SysTick_s &tick,
+    const PedalsSystemData_s &pedalsData,
+    const float wheel_rpms[])
+{
+
+    int16_t brake_torque_req = pedalsData.regenPercent * MAX_REGEN_TORQUE * -1;
```

Comment:
@Dopp-IO I have made this torque request here be negative. I am confused though, because it looks like our simple torque controller is using a positive commanded torque when accel_request is < 0. Let me know which implementation is supposed to be correct, this or the one on line 41 of this file.

Reply:
there are some misnomers in the code that need to be fixed. All torque requests in code are positive (even in regen) because we're really setting the magnitude of torque the inverter is allowed to produce to reach the commanded speed setpoint. If the speed setpoint is below the car's current speed, actual wheel torque will be negative.